packages/DataGrid/src/DataGrid.Rows.js

```diff
@@ -118,7 +118,13 @@ export const HeaderRow = React.memo(({ data: RowData, columnIndex, style }) => {
   const { style: styleProps = {}, ...moreProps } = typeof headerProps === "function" ? headerProps({ header }) : {};
 
   return (
-    <CellHeader role="columnheader" style={{ ...style, ...styleProps }} {...moreProps} borderType={borderType}>
+    <CellHeader
+      role="columnheader"
+      style={{ ...style, ...styleProps }}
+      {...moreProps}
+      borderType={borderType}
+      title={typeof header === "function" ? header() : header}
```

Comment:
since this is being used in more than one place now, can you create a variable for this?